web3_utils/load_public_keys_from_files.py

```diff
@@ -8,9 +15,28 @@ def load_public_keys_from_files(type: str, dir_path: str = path.join(getcwd(), "
     for file in files:
         if file.startswith(type):
             file_path = path.join(dir_path, file)
-
-            with open(file_path, "r") as f:
-                lines = [line.strip() for line in f if line.strip()]
-                result.extend(lines)
+            public_keys = read_public_keys_from_file(file_path)
+            result.extend(public_keys)
 
     return result
+
+
+class PublicKeyFile(TypedDict):
+    public_keys: list[str]
+    file_name: str
+    type: str
+
+
+# Seperate function introduced that returns the pubkeys on a per file basis as to not break existing code that uses the previous function
+def load_public_key_files(type: str, dir_path: str = path.join(getcwd(), "tmp", "public_keys")) -> list[PublicKeyFile]:
```

Comment:
I think the return type is wrong. the `list` should be uppercase:
List[PublicKeyFile]: